### PR TITLE
UX improvements in the Backup page

### LINKED ISSUE #19285: [BO][IMPROVIMENT][BACKUP CONTROLLER]
1 - Go to BO > Advanced Params > Databases > Backups
2 - Create a backup

## Is your feature request related to a problem?

* 1 - **Done in #19286** Use icon download instead of eyes (to remain consistent with the other controllers (see SQL management for example)
backup-icon

* 2 - **Done in #19286** Add a button to return to the list after download

* 3 - click on the download button

back-to-list

* 4 - Add filter by backup date

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Some improvement in admin backup controller
| Type?         | improvement 
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/19285 (1 & 2)
| How to test?  | See https://github.com/PrestaShop/PrestaShop/issues/19285

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19286)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/download_view.html.twig [hunk 1]
```diff
@@ -41,9 +41,16 @@
                 {{ 'Beginning the download ...'|trans({}, 'Admin.Advparameters.Notification') }}
               </p>
             </div>
-            <p>{{ 'Backup files should automatically start downloading.'|trans({}, 'Admin.Advparameters.Notification') }}</p>
-            <p class="mb-0">{{ 'If not,[1][2] please click here[/1]!'|trans({'[1]': ' <a href="'~ downloadFile.url ~'" class="btn btn-outline-primary btn-sm">', '[/1]': '</a> ', '[2]': '<i class="icon-download"></i>'}, 'Admin.Advparameters.Notification')|raw }}</p>
-
+            <p>
+				{{ 'Backup files should automatically start downloading.'|trans({}, 'Admin.Advparameters.Notification') }}
+				{{ 'If not,[1][2] please click here[/1]!'|trans({'[1]': ' <a href="'~ downloadFile.url ~'" class="btn btn-outline-primary btn-sm">', '[/1]': '</a> ', '[2]': '<i class="icon-download"></i>'}, 'Admin.Advparameters.Notification')|raw }}
+			</p>
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/download_view.html.twig]:
```suggestion
            <p>
              {{ 'Backup files should automatically start downloading.'|trans({}, 'Admin.Advparameters.Notification') }}
              {{ 'If not,[1][2] please click here[/1]!'|trans({'[1]': ' <a href="'~ downloadFile.url ~'" class="btn btn-outline-primary btn-sm">', '[/1]': '</a> ', '[2]': '<i class="icon-download"></i>'}, 'Admin.Advparameters.Notification')|raw }}
          </p>
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/download_view.html.twig [hunk 2]
```diff
@@ -41,9 +41,16 @@
...
+			<p class="mb-0">
+				<a class="btn btn-outline-primary btn-sm" href="{{ path('admin_backups_index') }}">
+					<i class="material-icons">keyboard_backspace</i>
+					{{ 'Back to list'|trans({}, 'Admin.Actions') }}
+				</a>
+			</p>
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/download_view.html.twig]:
```suggestion
          <p class="mb-0">
            <a class="btn btn-outline-primary btn-sm" href="{{ path('admin_backups_index') }}">
              <i class="material-icons">keyboard_backspace</i>
              {{ 'Back to list'|trans({}, 'Admin.Actions') }}
            </a>
          </p>
```

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PululuK** [comment]
Need feedback from @PrestaShop/prestashop-product-team. See #19285 !

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
PM ✅  => let's go for QA !

**Review by Junebyun — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
LGTM Thanks @PululuK !!

**Comment by PierreRambaud** [comment]
Thanks @PululuK

**Pull request merged**